app/src/main/java/org/sopt/havit/ui/contents/ContentsMoreFragment.kt

```diff
@@ -78,6 +76,15 @@ class ContentsMoreFragment(
         }
     }
 
+    // 이전 뷰에서 bundle로 보낸 content 데이터 받아오기
+    private fun setData() {
+        contentsData = arguments?.getParcelable(CONTENTS_MORE_DATA)!!
```

Comment:
```suggestion
        contentsData = arguments?.getParcelable(CONTENTS_MORE_DATA) ?: throw IllegalStateException()
        // 혹은
        contentsData = requireNotNull(arguments?.getParcelable(CONTENTS_MORE_DATA))
```

이렇게 널처리 이쁘게 해주면 좋겠죠!

Reply:
오옹 수정하겠습니당!